sdk/nodejs/api/client.gen.ts

```diff
@@ -219,26 +254,36 @@ export type DirectoryPipelineOpts = {
 export type DirectoryWithDirectoryOpts = {
   /**
    * Exclude artifacts that match the given pattern.
-   * (e.g. ["node_modules/", ".git*"]).
    */
   exclude?: string[]
 
   /**
    * Include only artifacts that match the given pattern.
-   * (e.g. ["app/", "package.*"]).
```

Comment:
why did this disappear?

Reply:
See https://github.com/dagger/dagger/pull/4367#issuecomment-1382619290